product_docs/docs/pem/9/upgrading/upgrading_pem_installation/upgrading_pem_installation_linux_rpm.mdx

```diff
@@ -8,105 +8,89 @@ redirects:
 
-Where `<X>` is the PostgreSQL or EDB Postgres Advanced Server version whose `libs` package you want to install.
+After upgrading the PEM server using yum or the `dnf` command, you must configure the PEM server. For detailed information, see [Configuring the PEM server](#configuring-the-pem-server).
 
-### Prerequisites to upgrade a PEM installation on a Debian or Ubuntu host
+### On a Debian or Ubuntu host
 
-To upgrade packages on a Debian or Ubuntu host:
+You can use the `apt-get` package manager to upgrade the installed version of the PEM server on supported versions of Debian or Ubuntu:
 
 ```shell
-apt-get update
-apt-get upgrade
+apt-get upgrade edb-pem
 ```
 
-To upgrade Advanced Server libs:
+After upgrading the PEM server with `apt-get`, you need to configure the PEM server. For detailed information, see [Configuring the PEM server](#configuring-the-pem-server).
 
-```shell
-apt-get install edb-as<X>-server-libs
-```
+### On a SLES host
 
-To upgrade PostgreSQL libs:
+You can use the zypper package manager to upgrade the installed version of the PEM Server on supported versions of a SLES host:
 
 ```shell
-apt-get install postgresql<X>-libs
+zypper update edb-pem
 ```
 
-Where `<X>` is the PostgreSQL or EDB Postgres Advanced Server version whose `libs` package you want to install.
+After upgrading the PEM server using zypper, you need to configure the PEM server. For detailed information, see [Configuring the PEM server](#configuring-the-pem-server).
 
-### Prerequisites to upgrade a PEM installation on a SLES host
+!!! Note
+    If you upgrade the PEM backend database server and the PEM server, update the `PG_INSTALL_PATH` and `DB_UNIT_FILE` parameters pointing to the new version in the `/usr/edb/pem/share/.install-config` file before you run the configure script.
 
-To upgrade packages on a SLES host:
+## Configuring the PEM server
+
+After upgrading the PEM server, you can use the following command to configure the PEM server:
 
 ```shell
-zypper update
-zypper upgrade
+/usr/edb/pem/bin/configure-pem-server.sh
 ```
 
-To upgrade EDB Postgres Advanced Server libs:
+The configure script uses the values from the old PEM server configuration file while running the script.
 
-```shell
-zypper install edb-as<x>-server-libs
-```
 
-To upgrade PostgreSQL libs:
+After executing the PEM server configuration file, use your version-specific service control command to restart the httpd service.
 
-```shell
-zypper install postgresql<x>-libs
-```
+For detailed information, see [Configuring the PEM server on Linux platforms](/pem/latest/installing/configuring_the_pem_server_on_linux/).
 
```

Comment:
Need to add a note here:-
Note:- In case the configure script gets hanged in between, Please identify any agent with an `alert_thread` > 0 in their configuration file, stop that Agent and retry running the configure script.
If still the problm persists Execute the below query and retry running the configure script.
SELECT pg_terminate_backend(pid) from pg_stat_activity where query='SELECT pem.process_one_alert()'